package.json

```diff
@@ -45,6 +45,7 @@
       "supabase"
     ],
     "overrides": {
+      "@radix-ui/react-dialog": "1.1.13",
```

Comment:
There are some libraries depends on @radix-ui/react-dialog, such as cmdk, vaul and fumadocs-ui.
To avoid version different, it need to be pinned.